Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -158,6 +169,96 @@ WarpX::computePhi (const amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho,
                    Real const required_precision,
                    int const max_iters) const
 {
```

Comment:
At this point, there is little overlap between the RZ code path and Cartesian code path for this function (`WarpX::computePhi`). 
Therefore, I am wondering whether we should split this into 2 functions (`WarpX::computePhiCartesian` and `WarpX::computePhiCylindrical`) for better readibility. (since it is sometimes easy to overlook, or get confused with, the `#ifdef`)

Reply:
I had the same thought. I'll do the split.